web/templates/pair/index.html.eex

```diff
@@ -23,9 +23,11 @@
 </ul>
 <%= if !past_week?(@year, @week) do %>
   <%= form_for @conn, pair_path(@conn, :delete, @year, @week), [as: :repairify, method: "delete"], fn _ -> %>
-    <div class="form-group">
-      <%= submit "Repairify", class: "btn btn-danger" %>
-    </div>
+    <%= if @conn.assigns.current_user.is_admin do %>
```

Comment:
In the future, I'll probably extract this logic out to the view to account for GroupRoles (which are not yet a thing).